app/views/layouts/_header.html.erb

```diff
@@ -2,7 +2,8 @@
   <%= header.with_navigation_item(text: current_user.name) %>
   <% if current_user.can_manage_others %>
     <%= header.with_navigation_item(text: t('.manage_users'), href: admin_manage_users_root_path) %>
-  <% elsif current_user.supervisor? && FeatureFlags.basic_user_roles.enabled? %>
+  <%end%>
+  <% if (current_user.supervisor? && FeatureFlags.basic_user_roles.enabled?) || FeatureFlags.allow_user_managers_service_access.enabled? %>
```

Comment:
This is a lot of logic to have a in a view - I think we could wrap it in a method to encapsulate it and make the view more readable 👍

Reply:
We did discuss this and I agree, however the `basic_user_roles` feature flag is not going to remain and will be removed when the user roles work goes live so it didn't feel necessary in the long run to encapsulate it into it's own method...  happy to add a method however